.github/workflows/standalone-fleet-ci.yml

```diff
@@ -0,0 +1,119 @@
+name: E2E Standalone Fleet
+
+on:
+  schedule:
+    - cron:  '0 5 30 * *'
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
+        description: 'Run the build with tmate debugging enabled'
+        required: false
+        default: "false"
+  pull_request:
+    paths-ignore:
+    - 'docs/**'
+    - 'charts/**'
+    - 'scripts/**'
+    - '*.md'
+
+env:
+  GOARCH: amd64
+  CGO_ENABLED: 0
+  SETUP_GO_VERSION: '^1.18'
+
+jobs:
+  e2e-fleet-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      -
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      -
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.SETUP_GO_VERSION }}
+      -
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      -
+        name: Install Ginkgo CLI
+        run: go install github.com/onsi/ginkgo/v2/ginkgo@v2.1
+      -
+        name: Build fleet binaries
+        run: |
+          go build -o bin/fleetcontroller-linux-$GOARCH ./cmd/fleetcontroller
+
+          go build -o "bin/fleet-linux-$GOARCH"
+          go build -o "bin/fleetagent-linux-$GOARCH" ./cmd/fleetagent
```

Comment:
we should be using `./scripts/build`: https://github.com/rancher/fleet/blob/master/scripts/build

This would ensure the link flags are provided and would mimic our build process: https://github.com/rancher/fleet/blob/d45a36738d276ef9caf84aecc850b69d66b6c2cf/scripts/build#L10-L13

Reply:
I don't like to call scripts from workflows, especially if they were built for another purpose or have too many arguments and if clauses.

For tests we don't want those linker flags. If this was a QA test for a release candidate, sure, we might want to set a version, strip debug info and symbols from the binary and such.

This script is also used by Drone, so any changes we might want to do will affect the release process of fleet. I wanted to avoid  that for now.